deploy/example/azschedulerextender/csi-azuredisk-azschedulerextender.yaml

```diff
@@ -0,0 +1,137 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: azschedulerextender
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: azschedulerextender-cluster-admin
+  namespace: kube-system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - kind: ServiceAccount
+    namespace: kube-system
+    name: azschedulerextender
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: azschedulerextender-config
+  namespace: kube-system
+data:
+  config.yaml: |
+    apiVersion: kubescheduler.config.k8s.io/v1alpha1
+    kind: KubeSchedulerConfiguration
+    schedulerName: azschedulerextender
+    algorithmSource:
+      policy:
+        configMap:
+          namespace: kube-system
+          name: azschedulerextender-policy
+    leaderElection:
+      leaderElect: true
+      lockObjectName: azschedulerextender
+      lockObjectNamespace: kube-system
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: azschedulerextender-policy
+  namespace: kube-system
+data:
+ policy.cfg : |
+  {
+    "kind" : "Policy",
+    "apiVersion" : "v1",
+    "predicates" : [
+      {"name" : "PodFitsHostPorts"},
+      {"name" : "PodFitsResources"},
+      {"name" : "NoDiskConflict"},
+      {"name" : "MatchNodeSelector"},
+      {"name" : "HostName"}
+    ],
+    "priorities" : [
+      {"name" : "LeastRequestedPriority", "weight" : 1},
+      {"name" : "BalancedResourceAllocation", "weight" : 1},
+      {"name" : "ServiceSpreadingPriority", "weight" : 1},
+      {"name" : "EqualPriority", "weight" : 1}
+    ],
+    "extenders" : [{
+      "urlPrefix": "http://localhost:8080",
```

Comment:
> `"urlPrefix": "http://localhost:8080"`

Should we use a non-root path to facilitate port sharing by multiple containers in the same pod, e.g. `"https://localhost:8080/api/mandalay/scheduler"`?

Reply:
Done. Let me know if we should change the name. I thought we were going to avoid mandalay?

Reply:
Yes, we should avoid "mandalay". I just couldn't think of anything useful in the moment. I am fine with the current name.